lib/dnsimple/client/vanity_name_servers.rb

```diff
@@ -0,0 +1,43 @@
+module Dnsimple
+  class Client
+    module VanityNameServers
+
+      # Enable vanity name servers for a domain. 
+      #
+      # @see https://developer.dnsimple.com/nameservers/vanity-nameservers/#enable
```

Comment:
The updated link is https://developer.dnsimple.com/v1/nameservers/vanity-nameservers/#enable